next.config.js

```diff
@@ -127,11 +128,12 @@ const withPWA = require('next-pwa')({
   importScripts: ['/pwa.js'],
   cacheStartUrl: false,
   cacheOnFrontEndNav: true,
-  skipWaiting: false
-})
+  skipWaiting: false,
+});
 const { loadEnvConfig } = require('@next/env');
-loadEnvConfig(".")
-module.exports = withPWA(nextConfig);
+loadEnvConfig('.');
+// module.exports = withPWA(nextConfig);
```

Comment:
this shouldn't be committed. Did you intend to use this for local testing? You can also turn the service work offline from your browser -> debug console -> applications -> service worker -> disable